cluster/photon-controller/util.sh

```diff
@@ -560,11 +560,20 @@ function wait-minion-apis {
 #
 function setup-pod-routes {
     local node
+
     KUBE_NODE_BRIDGE_NETWORK=()
     for (( node=0; node<${#NODE_NAMES[@]}; node++)); do
+
+        # This happens in two steps (wait for an address, wait for a non 172.x.x.x address)
+        # because it's both simpler and more clear what's happening.
         try-until-success-ssh ${KUBE_NODE_IP_ADDRESSES[$node]} \
-            "Waiting for cbr0 bridge on ${NODE_NAMES[$node]}"  \
-            'sudo ifconfig cbr0 | grep -oP "inet addr:\K\S+"'
+            "Waiting for cbr0 bridge on ${NODE_NAMES[$node]} to have an address"  \
+            'sudo ifconfig cbr0  | grep -oP "inet addr:\K\S+"'
+
+        try-until-success-ssh ${KUBE_NODE_IP_ADDRESSES[$node]} \
+            "Waiting for cbr0 bridge on ${NODE_NAMES[$node]} to have correct address"  \
+            'sudo ifconfig cbr0  | grep -oP "inet addr:\K\S+" | grep -v  "^172."'
```

Comment:
Isn't it very hacky way to fix it? Is there any other way we can figure out the real network?